osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs

```diff
@@ -44,8 +30,8 @@ private void layout()
                     EdgeEffect = new EdgeEffect
                     {
                         Type = EdgeEffectType.Glow,
-                        Radius = 20,
-                        Colour = glowColour,
+                        Radius = 10,
+                        Colour = Color4.Gray.Opacity(0.2f),

```

Comment:
Or define a new SpinnerTickGlow colour like there is with SpinnerBase and SpinnerFill with the default being `SpinnerTickGlow = Gray(128)`, or perhaps even referencing one of the existing grays, `OsuColour.Gray8` is close enough imho.